src/services/customer.ts

```diff
@@ -183,4 +186,56 @@ export default class CustomerService extends Service {
 
         return response
     }
+
+    /**
+     * Create new Customer/Account under Master Account
+     * @ref https://developers.google.com/google-ads/api/reference/rpc/v3/CustomerService method CreateCustomerClient
+     * @param {CreateCustomerOptions} options
+     * @param {CreateCustomerFlowSettings} [flow_settings]
+     * @param {boolean} [flow_settings.return_customer] Optional flag set to False by default. Provide True if you
+     *   prefer to have Customer instance as result instead of CreateCustomerClientResponse
+     */
+    public async createCustomerClient(
+        options: CreateCustomerOptions,
+        flow_settings: CreateCustomerFlowSettings = { return_customer: false }
+    ): CreateCustomerResponse {
+        if (flow_settings && flow_settings.return_customer && !flow_settings.customer_options) {
+            throw new TypeError(`Missing 'customer_options' in 'flow_settings'. It should be an object with: 'login_customer_id' and 'refresh_token'`)
+        }
+
+        const request = new grpc.CreateCustomerClientRequest();
+
+        const customerClientPB = this.buildResource('Customer', options.customer_client) as grpc.Customer
+
+        request.setCustomerId(options.customer_id)
+        request.setCustomerClient(customerClientPB)
+
+        if (options.access_role) {
+            request.setAccessRole(options.access_role)
+        }
+
+        if (options.email_address) {
+            const emailValue = new StringValue()
+
+            emailValue.setValue(options.email_address)
+
+            request.setEmailAddress(emailValue)
+        }
+
+        const response = await this.serviceCall(
+                'createCustomerClient',
+                request,
+            ) as CreateCustomerClientResponse
+
+        if (!flow_settings || !flow_settings.return_customer) {
+            return response
+        }
+
+        const customer_id = (response.resource_name as string).split('/')[1]
+
+        return flow_settings.gads_api_client.Customer({
+            customer_account_id: customer_id,
+            ...flow_settings.customer_options
```

Comment:
I think all of this data (cid, login customer id etc.) is available in `super.client` (currently private, but maybe a method could be added to return the client details in `grpc.ts`. That means the only additional argument required would be `{ return_customer }`.

Reply:
I checked it, and seems not that easy. 
We provide refresh_token and login_customer_id only when we call GoogleAdsApi.Customer method, there we create GRPC client by passing this properties to it from provided CustomerOptions.
`client` from GRPC class is not the one we can use in this flow, but it is of class GoogleAdsClient, and it doesn't have `.Customer` method, in fact it has 'getService` but as far as I can see it's related to Google Ads Api GRPC services. So I cannot easily use this client and still will need to have access to GoogleAdsApi.Customer.
I exposed refreesh_token from GRPC client, and for `login_customer_id` I used customer_id passed as options to `createCustomer` method, because here `customer_id` is master account id.
@kritzware check please changes once you have time. Thank you.

Reply:
Okay, thanks for addressing those earlier comments. I've just left two comments, and then we should be ready to merge this in!